src/main/java/com/ericsson/gerrit/plugins/eiffel/events/models/KeyValue.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Consider if "CustomData" is a better name for this class.

Reply:
It's not really customdata. These are Key/value fields inside customdata which itself is just an array in the data structure.

So it will basically be used like CustomData.add(KeyValue)